@extends('layouts.rna')

@section('head') @parent
<link rel="stylesheet" type="text/css" href="{{ URL::asset('/css/wait.css') }}" media="all" />
@endsection

@section('content-main')
<div class='box'>
   <div id=header1>2015</div><div id=content>
      <p>Chen, C., Xiao, Y., & Huang, Y. (2015). Improving the replica-exchange molecular-dynamics method for efficient sampling in the temperature space. Phys Rev E Stat Nonlin Soft Matter Phys, 91(5), 052708. doi: 10.1103/PhysRevE.91.052708
      </p>
      <p>Wang, J., Zhao, Y., Zhu, C., & Xiao, Y. (2015). 3dRNAscore: a distance and torsion angle dependent evaluation function of 3D RNA structures. Nucleic Acids Res, 43(10), e63. doi: 10.1093/nar/gkv141

      </p>
      <p>Wang, J., Zhao, Y., Wang, J., & Xiao, Y. (2015). Computational study of stability of an H-H-type pseudoknot motif. Phys Rev E Stat Nonlin Soft Matter Phys, 92(6), 062705. doi: 10.1103/PhysRevE.92.062705

      </p>
      <p>Wang, E., Wang, J., Chen, C., & Xiao, Y. (2015). Computational evidence that fast translation speed can increase the probability of cotranslational protein folding. Sci Rep, 5, 15316. doi: 10.1038/srep15316

      </p>

   </div>
   <div id=header1>2014</div><div id=content>
      <p>Zhou, R., Maisuradze, G. G., Sunol, D., Todorovski, T., Macias, M. J., Xiao, Y., Scheraga, H. A., Czaplewski, C. & Liwo, A. (2014). Folding kinetics of WW domains with the united residue force field for bridging microscopic motions and experimental measurements. Proc Natl Acad Sci U S A 111, 18243-8.</p>
      <p>Gong, Z., Zhao, Y. J., Chen, C. J., Duan, Y. & <b>Xiao, Y</b>. (2014). Insights into Ligand Binding to PreQ(1) Riboswitch Aptamer from Molecular Dynamics Simulations. Plos One 9.</p>
      <p>Chen, C. J., Huang, Y. Z., Jiang, X. W. & <b>Xiao, Y</b>. (2014). A fast tomographic method for searching the minimum free energy path. Journal Of Chemical Physics 141.</p>
   </div>
   <div id=header1>2013</div><div id=content>
      <p>Zhao, Y., Wang, J., Chen, X. W., Luo, H. T., Zhao, Y. J., <b>Xiao, Y</b>. & Chen, R. S. (2013). Large-scale study of long non-coding RNA functions based on structure and expression features. Science China-Life Sciences 56, 953-959.</p>
      <p>Li, L., Huang, Y. Z. & <b>Xiao, Y</b>. (2013). How to Use Not-Always-Reliable Binding Site Information in Protein-Protein Docking Prediction. Plos One 8.</p>
      <p>Huang, Y. Y., Liu, S. Y., Guo, D. C., Li, L. & <b>Xiao, Y</b>. (2013). A novel protocol for three-dimensional structure prediction of RNA-protein complexes. Scientific Reports 3.</p>
      <p>Guo, D. C., Liu, S. Y., Huang, Y. Y. & <b>Xiao, Y</b>. (2013). Preorientation of protein and RNA just before contacting. Journal Of Biomolecular Structure & Dynamics 31, 716-728.</p>
      <p>Chen, C. J., Wang, E. C., Liu, P. Y. & <b>Xiao, Y</b>. (2013). Simulation study of the role of the ribosomal exit tunnel on protein folding. Physical Review E 87.</p>
      <p>Chen, C. J., Huang, Y. Z. & <b>Xiao, Y</b>. (2013). Enhanced sampling of molecular dynamics simulation of peptides and proteins by double coupling to thermal bath. Journal Of Biomolecular Structure & Dynamics 31, 206-214.</p>
      <p>Chen, C. J., Huang, Y. Z., Jiang, X. W. & <b>Xiao, Y</b>. (2013). Binding free-energy calculation of an ion-peptide complex by constrained dynamics. Physical Review E 87.</p>
      <p>Chen, C. J., Huang, Y. Z., Ji, X. F. & <b>Xiao, Y</b>. (2013). Efficiently finding the minimum free energy path from steepest descent path. Journal Of Chemical Physics 138.</p>
   </div>
   <div id=header1>2012</div><div id=content>
      <p>Zhao, Y., Huang, Y., Gong, Z., Wang, Y., Man, J. & <b>Xiao, Y</b>. (2012). Automated and fast building of three-dimensional RNA structures. Sci Rep 2, 734.</p>
      <p>Maisuradze, G. G., Zhou, R., Liwo, A., <b>Xiao, Y</b>. & Scheraga, H. A. (2012). Effects of Mutation, Truncation, and Temperature on the Folding Kinetics of a WW Domain. Journal Of Molecular Biology 420, 350-365.</p>
      <p>Jiang, X. W., Zhong, A. H., Chen, C. J., Huang, Y. Z. & <b>Xiao, Y</b>. (2012). Network approach to identify the folding transition states of peptides and proteins. Physical Review E 86.</p>
      <p>Gong, Z., Zhao, Y., Chen, C. & <b>Xiao, Y</b>. (2012). Computational study of unfolding and regulation mechanism of preQ1 riboswitches. PLoS One 7, e45239.</p>
      <p>Chen, C., Huang, Y. & <b>Xiao, Y</b>. (2012). Free-energy calculations along a high-dimensional fragmented path with constrained dynamics. Phys Rev E Stat Nonlin Soft Matter Phys 86, 031901.</p>
   </div>
   <div id=header1>2011</div><div id=content>
      <p>Zhou, R., He, Y. & <b>Xiao, Y</b>. (2011). Multi-nucleation and vectorial folding pathways of large helix protein. Computational Biology And Chemistry 35, 169-173.</p>
      <p>Zhao, Y. J., Gong, Z. & <b>Xiao, Y</b>. (2011). Improvements of the Hierarchical Approach for Predicting RNA Tertiary Structure. Journal Of Biomolecular Structure & Dynamics 28, 815-826.</p>
      <p>Li, L., Guo, D. C., Huang, Y. Y., Liu, S. Y. & <b>Xiao, Y</b>. (2011). ASPDock: protein-protein docking algorithm using atomic solvation parameters model. Bmc Bioinformatics 12.</p>
      <p>Lei, H. X., Chen, C. J., <b>Xiao, Y</b>. & Duan, Y. (2011). The protein folding network indicates that the ultrafast folding mutant of villin headpiece subdomain has a deeper folding funnel. Journal Of Chemical Physics 134.</p>
      <p>Gong, Z., Zhao, Y. J., Chen, C. J. & <b>Xiao, Y</b>. (2011). Role of Ligand Binding in Structural Organization of Add A-riboswitch Aptamer: A Molecular Dynamics Simulation. Journal Of Biomolecular Structure & Dynamics 29, 403-416.</p>
      <p>Fleishman, S. J., Whitehead, T. A., Strauch, E. M., Corn, J. E., Qin, S. B., Zhou, H. X., Mitchell, J. C., Demerdash, O. N. A., Takeda-Shitaka, M., Terashi, G., Moal, I. H., Li, X. F., Bates, P. A., Zacharias, M., Park, H., Ko, J. S., Lee, H., Seok, C., Bourquard, T., Bernauer, J., Poupon, A., Aze, J., Soner, S., Ovali, S. K., Ozbek, P., Ben Tal, N., Haliloglu, T., Hwang, H., Vreven, T., Pierce, B. G., Weng, Z. P., Perez-Cano, L., Pons, C., Fernandez-Recio, J., Jiang, F., Yang, F., Gong, X. Q., Cao, L. B., Xu, X. J., Liu, B., Wang, P. W., Li, C. H., Wang, C. X., Robert, C. H., Guharoy, M., Liu, S. Y., Huang, Y. Y., Li, L., Guo, D. C., Chen, Y., <b>Xiao, Y</b>., London, N., Itzhaki, Z., Schueler-Furman, O., Inbar, Y., Potapov, V., Cohen, M., Schreiber, G., Tsuchiya, Y., Kanamori, E., Standley, D. M., Nakamura, H., Kinoshita, K., Driggers, C. M., Hall, R. G., Morgan, J. L., Hsu, V. L., Zhan, J., Yang, Y. D., Zhou, Y. Q., Kastritis, P. L., Bonvin, A. M. J. J., Zhang, W. Y., Camacho, C. J., Kilambi, K. P., Sircar, A., Gray, J. J., Ohue, M., Uchikoga, N., Matsuzaki, Y., Ishida, T., Akiyama, Y., Khashan, R., Bush, S., Fouches, D., Tropsha, A., Esquivel-Rodriguez, J., Kihara, D., Stranges, P. B., Jacak, R., Kuhlman, B., Huang, S. Y., Zou, X. Q., Wodak, S. J., Janin, J. & Baker, D. (2011). Community-Wide Assessment of Protein-Interface Modeling Suggests Improvements to Design Methodology. Journal Of Molecular Biology 414, 289-302.</p>
   </div>
   <div id=header1>2010</div><div id=content>
      <p>Wang, G. Y., Du, C. H., Chen, H., Simha, R., Rong, Y. W., <b>Xiao, Y</b>. & Zeng, C. (2010). Process-based network decomposition reveals backbone motif structure. Proceedings Of the National Academy Of Sciences Of the United States Of America 107, 10478-10483.</p>
      <p>Jiang, X. W., Chen, C. J. & <b>Xiao, Y</b>. (2010). Improvements of Network Approach for Analysis of the Folding Free-Energy Surface of Peptides and Proteins. Journal Of Computational Chemistry 31, 2502-2509.</p>
      <p>Gong, Z., Zhao, Y. J. & <b>Xiao, Y</b>. (2010). RNA Stability Under Different Combinations of Amber Force Fields and Solvation Models. Journal Of Biomolecular Structure & Dynamics 28, 431-441.</p>
      <p>Feng, J. H., Li, M. F., Huang, Y. Z. & <b>Xiao, Y</b>. (2010). Symmetric Key Structural Residues in Symmetric Proteins with Beta-Trefoil Fold. Plos One 5.</p>
      <p>Chen, C. J. & <b>Xiao, Y</b>. (2010). Accurate Free Energy Calculation along Optimized Paths. Journal Of Computational Chemistry 31, 1368-1375.</p>
   </div>
   <div id=header1>2009</div><div id=content>
      <p><b>Xiao, Y</b>., Chen, C. J. & He, Y. (2009). Folding Mechanism of Beta-Hairpin Trpzip2: Heterogeneity, Transition State and Folding Pathways. International Journal Of Molecular Sciences 10, 2838-2848.</p>
      <p>Li, M., Huang, Y. & <b>Xiao, Y</b>. (2009). A method for identification of selenoprotein genes in archaeal genomes. Genomics Proteomics Bioinformatics 7, 62-70.</p>
      <p>He, Y., Zhou, R., Huang, Y. & <b>Xiao, Y</b>. (2009). Foldable subunits of helix protein. Comput Biol Chem 33, 325-8.</p>
      <p>He, Y., <b>Xiao, Y</b>., Liwo, A. & Scheraga, H. A. (2009). Exploring the parameter space of the coarse-grained UNRES force field by random search: selecting a transferable medium-resolution force field. J Comput Chem 30, 2127-35.</p>
      <p>He, Y., Chen, C. & <b>Xiao, Y</b>. (2009). United-residue (UNRES) Langevin dynamics simulations of trpzip2 folding. J Comput Biol 16, 1719-30.</p>
      <p>Chen, H., Huang, Y. & <b>Xiao, Y</b>. (2009). A simple method of identifying symmetric substructures of proteins. Comput Biol Chem 33, 100-7.</p>
   </div>
   <div id=header1>2008</div><div id=content>
      <p>Wang, X., Huang, Y. & <b>Xiao, Y</b>. (2008). Structural-symmetry-related sequence patterns of the proteins of beta-propeller family. J Mol Graph Model 26, 829-33.</p>
      <p>Li, M., Huang, Y. & <b>Xiao, Y</b>. (2008). Effects of external interactions on protein sequence-structure relations of beta-trefoil fold. Proteins 72, 1161-70.</p>
      <p>Chen, C. & <b>Xiao, Y</b>. (2008). Observation of multiple folding pathways of beta-hairpin trpzip2 from independent continuous folding trajectories. Bioinformatics 24, 659-65.</p>
   </div>
   <div id=header1>2007</div><div id=content>
      <p>Ji, X., Chen, H. & <b>Xiao, Y</b>. (2007). Hidden symmetries in the primary sequences of beta-barrel family. Comput Biol Chem 31, 61-3.</p>
      <p>Huang, Y. & <b>Xiao, Y</b>. (2007). Detection of gene duplication signals of Ig folds from their amino acid sequences. Proteins 68, 267-72.</p>
      <p>Chen, C., Li, L. & <b>Xiao, Y</b>. (2007). All-atom contact potential approach to protein thermostability analysis. Biopolymers 85, 28-37.</p>
   </div>
   <div id=header1>2006</div><div id=content>
      <p>Zhang, S. & <b>Xiao, Y</b>. (2006). Quasiperiodic property in Alu repeats. Phys Rev E Stat Nonlin Soft Matter Phys 74, 022901.</p>
      <p>Chen, C. J., Li, L. & <b>Xiao, Y</b>. (2006). Identification of key residues in proteins by using their physical characters. Physical Review E 73.</p>
      <p>Chen, C. & <b>Xiao, Y</b>. (2006). Molecular dynamics simulations of folding processes of a beta-hairpin in an implicit solvent. Phys Biol 3, 161-71.</p>
   </div>
   <div id=header1>2005</div><div id=content>
      <p>Chen, C. J., <b>Xiao, Y</b>. & Zhang, L. S. (2005). A directed essential dynamics simulation of peptide folding. Biophysical Journal 88, 3276-3285.</p>
   </div>
   <div id=header1>2003</div><div id=content>
      <p><b>Xiao, Y</b>., Huang, Y. Z., Li, M. F., Xu, R. Z. & Xiao, S. F. (2003). Nonlinear analysis of correlations in Alu repeat sequences in DNA. Physical Review E 68.</p>
   </div>
</div>
@endsection

